src/main/java/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import service.FileWriter;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String report, String fileName) {
```

Comment:
The method write should not use a hardcoded file path. It's better to accept the path as a parameter, which you are already doing, but ensure you are using a relative path that works in any environment.